### PR TITLE
Implement customization for bookmark capitalization

### PR DESCRIPTION
**Pull request by 5brian**
# Implement customization for bookmark capitalization

This pull request adds functionality to allow users to customize how bookmarks are capitalized

## Changes made:
- Added new user setting for bookmark capitalization preference -- Capitalized, Lowercase, and Default (same as saved name)
- Implemented logic to apply chosen capitalization style when saving/displaying bookmarks
- Updated UI to show capitalization options in settings menu

## Benefits:
- Gives users more control over how their bookmarks appear
- Improves consistency of bookmark display based on user preference
- Enhances overall customization options

## Testing:
- Manually tested different capitalization options

Please review and let me know if any changes or additional information is needed

### CODE REVIEW

File: src/components/app-link.imba [hunk 1]
```diff
@@ -21,7 +21,11 @@ tag app-link
 				css w:20px h:20px mr:10px rd:3px
 
 			<.name> link.display-name
-				css tt:capitalize fs:20px overflow-wrap:anywhere
+				css fs:20px overflow-wrap:anywhere
```
Comment (familyfriendlymikey) [inline src/components/app-link.imba:24]:
```imba
tt:{config.data.case}
```

**Comment by familyfriendlymikey** [comment]
Sure, I can merge this soon. Quite busy these days so feel free to bug me in a couple days if I forget.

**Comment by 5brian** [comment]
Okay, re-did the css line, thanks for reviewing :)

**Review by familyfriendlymikey — APPROVED** [review]

**Comment by familyfriendlymikey** [comment]
Nice work. It's on main, but I'm hesitant to publish a new version of the chrome plugin because I while ago I changed bang links to use a $0,$1,$2,etc. format versus just appending to the end of the link, and I wasn't confident that it would convert everyone's links as intended. But maybe I should just release and if people have problems deal with it then, since I don't have time to be super thorough.

**Pull request merged**